runtime/mainnet/Cargo.toml

```diff
@@ -36,6 +37,8 @@ frame-try-runtime.workspace = true
 pallet-aura.workspace = true
 pallet-authorship.workspace = true
 pallet-balances.workspace = true
+pallet-collective.workspace = true
+pallet-membership.workspace = true
```

Comment:
FYI - Could be addressed in a follow up, but I spiked a similar implementation without the membership pallet and it seems to work just fine for a small set, which I imagine we will be for the first foreseeable period. Simplifies the implementation.